README.md

```diff
@@ -86,22 +136,38 @@ module.exports = {
 ### Brief explanation of config
 
 - `<rootDir>` is a special syntax for root of your project (here by default it's project's root /)
+
 - we're using some `"globals"` to pass information about where our tsconfig.json file is that we'd like to be able to transform HTML files through ts-jest
+
 - `"transform"` – run every TS, JS, or HTML file through so called _preprocessor_ (we'll get there); this lets Jest understand non-JS syntax
+
 - `"testMatch"` – we want to run Jest on files that matches this glob
+
 - `"moduleFileExtensions"` – our modules are TypeScript and JavaScript files
+
 - `"moduleNameMapper"` – if you're using absolute imports here's how to tell Jest where to look for them; uses regex
+
 - `"setupFilesAfterEnv"` – this is the heart of our config, in this file we'll setup and patch environment within tests are running
+
 - `"transformIgnorePatterns"` – unfortunately some modules (like @ngrx) are released as TypeScript files, not pure JavaScript; in such cases we cannot ignore them (all node_modules are ignored by default), so they can be transformed through TS compiler like any other module in our project.
-- `"snapshotSerializers"` - array of serializers which will be applied to snapshot the code. Note: by default angular adds some angular-specific attributes to the code (like `ng-reflect-*`, `ng-version="*"`, `_ngcontent-c*` etc). This package provides serializer to remove such attributes. This makes snapshots cleaner and more human-readable. To remove such specific attributes use `AngularNoNgAttributesSnapshotSerializer` serializer. You need to add `AngularNoNgAttributesSnapshotSerializer` serializer manually (see [`test` app configuration](https://github.com/thymikee/jest-preset-angular/blob/master/e2e/test-app-v9/package.json#L47-L51)).
 
-## [AST Transformer](https://github.com/thymikee/jest-preset-angular/blob/master/src/InlineHtmlStripStylesTransformer.ts)
+- `"snapshotSerializers"` - array of serializers which will be applied to snapshot the code. Note: by default angular 
+adds some angular-specific attributes to the code (like `ng-reflect-*`, `ng-version="*"`, `_ngcontent-c*` etc). 
+This package provides serializer to remove such attributes. This makes snapshots cleaner and more human-readable. 
+To remove such specific attributes use `AngularNoNgAttributesSnapshotSerializer` serializer. 
+You need to add `AngularNoNgAttributesSnapshotSerializer` serializer manually (see [`test` app configuration](https://github.com/thymikee/jest-preset-angular/blob/master/e2e/test-app-v9/package.json#L47-L51)).
+
+
+## AST Transformers
+
+Jest doesn't run in browser nor through dev server. It uses jsdom to abstract browser environment. So we have to cheat 
+a little and inline our templates and get rid of styles (we're not testing CSS) because otherwise Angular will try 
+to make XHR call for our templates and fail miserably.
```

Comment:
Note: This part is also not correct anymore after the next update. The Angular Compiler does much more than just this using ast transformers.

Reply:
Ye, actually once we switch to use `replaceResources` transformer, we can remove this section.